pilot/pkg/serviceregistry/cloudfoundry/controller.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 Istio Authors
+// Copyright 2018 Istio Authors
```

Comment:
You shouldn't update old copyright notices. Istio authors holds the copyright from 2017 onwards.

Reply:
figured it I was touching in the new year it'd be more correct to move it forward.